src/lib.rs

```diff
@@ -1260,6 +1264,30 @@ mod test {
         Ok(())
     }
 
+    #[cfg(feature = "polars")]
```

Comment:
we can move all tests to polars_dataframe.rs so we don't need the cfg here

Reply:
Okay. I'll do it